### PR TITLE
Add tests for eosjs2-api.ts

### PR DESCRIPTION
**Pull request by randytorres**

### CODE REVIEW

**Comment by tbfleming** [comment]
Please remove the log file

File: tsconfig.node.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "module": "es2015",
+        "module": "commonjs",
```
Comment (jeffreyssmith2nd) [inline tsconfig.node.json]: See what @esheffield did in place of this. You can remove this change, just make sure your package.json merge is correct.

Comment (randytorres) [reply tsconfig.node.json]: Once his prs merged ill rebase and get rid of these changes i made.

Comment (esheffield) [reply tsconfig.node.json]: OK, my testing PRs have been merged.

**Comment by jeffreyssmith2nd** [comment]
I just merged my linting changes, so please run yarn lint before you merge. You can run yarn lint-fix to auto fix almost all issues.

**Pull request merged**